_posts/2014-02-15-geting-started-index.md

```diff
@@ -74,7 +74,7 @@ You'll also need to add the Sugar application to the `application/0` function in
 
 ```elixir
 def application do
-  [ applications: [ :sugar ],
+  [ applications: [],
```

Comment:
We should probably mention both styles, since I'm not sure if all of the Elixir/Mix versions we're supporting include the `extra_applications` feature.  Either way, I'm pretty sure the old way still requires the application to include `:sugar` in the applications list explicitly (if I'm correctly recalling various complications around the templates system).  Also, I suspect `mod` is unnecessary if the app is using `extra_applications` (or perhaps even if it isn't; I'll do some quick testing here to verify).